apps/zotonic_mod_copyright/README.md

```diff
@@ -0,0 +1,16 @@
+Copyrights
+===========
+
+This modules implements an admin panel and templates to select and view the copyrights
+attached to resources.  It also enables to add (legal) attribution to resources.
+
+This modules uses two sets of rights statements:
+
+ * https://creativecommons.org/licenses/#licenses-40
+ * https://rightsstatements.org/page/1.0/?language=en
+
+The properties manages are:
+
+ * `license`
+ * `copyright_attribution`
```

Comment:
I think the names of the properties were changed to `rights`, `rights_attribution` and, `rights_year`.

Reply:
Indeed - fixing that.